README.md

```diff
@@ -1,65 +1,73 @@
-README
-======
+<p align="center"><a href="https://symfony.com" target="_blank">
+    <img src="http://symfony.com/logos/symfony_black_02.svg">
```

Comment:
you should use a HTTPS URL here, even though it is not an issue on github as they proxy all images since a long time to avoid mixed content (and other risks)